tests/sdk/conftest.py

```
@@ -0,0 +1,73 @@
+import pytest
+import pytest_asyncio
+from fastapi.testclient import TestClient
+
+# Add the SDK src to the path to allow imports
+sdk_src_path = Path(__file__).parent.parent.parent / "sdks" / "python" / "src"
+sys.path.insert(0, str(sdk_src_path))
+
+# This is a bit of a hack to make the main conftest discoverable
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from sdks.python.src.honcho.async_client.client import AsyncHoncho  # noqa: E402
```

Ruff (0.11.9)

Unused `noqa` directive (unused: `E402`)

Remove unused `noqa` directive

(RUF100)